marlowe-apps/Finder.md

```diff
@@ -57,51 +53,35 @@ Available options:
-                           be set as the environment variable
-                           MARLOWE_RT_HISTORY_COMMAND_PORT (default: 3717)
-  --history-query-port PORT_NUMBER
-                           The port number of the history server's query API.
-                           Can be set as the environment variable
-                           MARLOWE_RT_HISTORY_QUERY_PORT (default: 3718)
-  --history-sync-port PORT_NUMBER
-                           The port number of the history server's
+                           (default: 13715)
+  --marlowe-sync-host HOST_NAME
+                           The hostname of the Marlowe Runtime marlowe-sync
+                           server. Can be set as the environment variable
+                           MARLOWE_RT_SYNC_HOST (default: "127.0.0.1")
+  --marlowe-sync-sync-port PORT_NUMBER
+                           The port number of the marlowe-sync server's
                            synchronization API. Can be set as the environment
-                           variable MARLOWE_RT_HISTORY_SYNC_PORT (default: 3719)
-  --discovery-host HOST_NAME
-                           The hostname of the Marlowe Runtime discovery server.
-                           Can be set as the environment variable
-                           MARLOWE_RT_DISCOVERY_HOST (default: "127.0.0.1")
-  --discovery-query-port PORT_NUMBER
-                           The port number of the discovery server's query API.
-                           Can be set as the environment variable
-                           MARLOWE_RT_DISCOVERY_QUERY_PORT (default: 3721)
-  --discovery-sync-port PORT_NUMBER
-                           The port number of the discovery server's
+                           variable MARLOWE_RT_SYNC_SYNC_PORT (default: 13724)
+  --marlowe-sync-header-port PORT_NUMBER
+                           The port number of the marlowe-sync server's header
                            synchronization API. Can be set as the environment
-                           variable MARLOWE_RT_DISCOVERY_SYNC_PORT
-                           (default: 3722)
+                           variable MARLOWE_RT_SYNC_HEADER_PORT (default: 13725)
+  --marlowe-sync-query-port PORT_NUMBER
+                           The port number of the marlowe-sync server's query
+                           API. Can be set as the environment variable
+                           MARLOWE_RT_SYNC_QUERY_PORT (default: 13726)
   --tx-host HOST_NAME      The hostname of the Marlowe Runtime transaction
                            server. Can be set as the environment variable
                            MARLOWE_RT_TX_HOST (default: "127.0.0.1")
   --tx-command-port PORT_NUMBER
                            The port number of the transaction server's job API.
                            Can be set as the environment variable
-                           MARLOWE_RT_TX_COMMAND_PORT (default: 3723)
+                           MARLOWE_RT_TX_COMMAND_PORT (default: 13723)
```

Comment:
How come all these default values got changed?

Reply:
It was my test environment, which has multiple testnets running on different sets of ports. I'll fix.

Reply:
Fixed in dd7cbd5b7646c2d03f5ee88274dab8653a21aba8.